docs/user/webapp/webapp.md

```diff
@@ -68,35 +75,40 @@ The web asks your for inputs TOSCAna needs to transform your CSAR.
 If a input field is highlighted red it means that it is required or that it is wrong.
 To get more information about an specific input you can hover over the info icon.
 
-**Important:** To get more information about the inputs check the documentation of the platform you want to transform to
```

Comment:
line 74: `the web asks your` -> `The web app asks you for inputs which are needed to transform the CSAR.`